scene/gui/label.cpp

```diff
@@ -654,7 +654,10 @@ void Label::_notification(int p_what) {
 		} break;
 
 		case NOTIFICATION_RESIZED: {
-			lines_dirty = true;
+			if (autowrap_mode != TextServer::AUTOWRAP_OFF || overrun_behavior != TextServer::OVERRUN_NO_TRIMMING) {
```

Comment:
This feels wrong because at the end of `_shape` we already call `update_minimum_size()` but for the opposite checks. Are you sure it's here where we're supposed to fix it?

Reply:
For these modes and behaviors, a change in size may result in a change in the minimum size. The minimum height varies due to the size's width in these modes.

https://github.com/godotengine/godot/blob/2f57da290796fbba7c141240bef22baf21615888/scene/gui/label.cpp#L676-L683

Under the opposite option, changing the size does not actually change the minimum size. Calling `update_minimum_size()` in `_shape()` does not seem necessary unless you just switched to these options.